lib/contentProfileListener.js

```diff
@@ -89,21 +108,33 @@ class ContentProfileImporter extends SHRContentProfileParserListener {
       // Create new rule, apply it in enterHeaderFlag function
       this._currentRule = new ContentProfileRule();
     }
+    else if (!ctx.headerFlags()) {
```

Comment:
Can't this just be a normal `else`?  It's the inverse of what you already said in `if`.

Reply:
Very true, fixed in https://github.com/standardhealth/shr-text-import/pull/106/commits/27603300c7219801657539ca83a166ce7217b755.